packages/manager/modules/pci/src/projects/project/inactive/inactive.html

```diff
@@ -0,0 +1,21 @@
+<oui-modal
+    on-dismiss="$ctrl.goBack()"
+    heading="{{ $ctrl.title }}"
+    type="warning"
+    primary-label="{{ ::($ctrl.project.hasPendingDebt()
+            ? ('pci_projects_project_pay_bill' | translate)
+            : ($ctrl.project.isSuspended() ? ('pci_projects_create_project' | translate) : '')
+        ) }}"
+    primary-action="$ctrl.project.hasPendingDebt() ? $ctrl.goToBilling() : $ctrl.goToNewProject()"
+    secondary-label="{{ ::(($ctrl.project.hasPendingDebt() && !$ctrl.project.isSuspended())
+            ? ('pci_projects_project_view' | translate)
+            : ((!$ctrl.project.hasPendingDebt() && $ctrl.project.isSuspended())
+                ? ('pci_projects_project_inactive_close' | translate)
+                : '')
+        ) }}"
```

Comment:
I think it's better to export all this logic inside a JS function:
```suggestion
    secondary-label="{{ ::$ctrl.getTradKey() | translate }}"
```

```
getTradKey() {
   if ($ctrl.project.hasPendingDebt() && !$ctrl.project.isSuspended()) {
      return 'pci_projects_project_view';
   } else if (!$ctrl.project.hasPendingDebt() && $ctrl.project.isSuspended()) {
      return 'pci_projects_project_inactive_close';
   }

   return '';
}
```

Reply:
Thanks @mohammed-zahaf for the review. Since we have the prod today, I will work on your comment later on, since it is too late now. Hope that is ok for you